src/internal/firstValueFrom.ts

```diff
@@ -13,6 +13,12 @@ import { Subscription } from './Subscription';
  * If the observable stream emits an error, the returned promise will reject
  * with that error.
  *
+ * **WARNING**: Only use this with observables you *know* will complete. If the source
```

Comment:
Hi there. Would this sentence be entirely true for `firstValueFrom`? There is a case where the source stream doesn't have to ever complete, it just have to emit at least one value. I would add ...`*know* will complete or emit at least once.`. Does this makes sense?